ray-operator/apis/ray/v1/raycluster_types.go

```diff
@@ -122,6 +123,12 @@ type RayClusterStatus struct {
 	MinWorkerReplicas int32 `json:"minWorkerReplicas,omitempty"`
 	// MaxWorkerReplicas indicates sum of maximum replicas of each node group.
 	MaxWorkerReplicas int32 `json:"maxWorkerReplicas,omitempty"`
+	// DesiredCPUs indicates total desired CPUs for the cluster
```

Comment:
TPUs are now also a native resource in Ray, should we add them here as well?

Reply:
Sounds good. What’s the resource name?

Reply:
`google.com/tpu`